buckets-mdapi/src/opts.rs

```diff
@@ -1,6 +1,5 @@
-/*
- * Copyright 2019 Joyent, Inc.
- */
+// Copyright 2019 Joyent, Inc.
+// Copyright 2023 MNX Cloud, Inc.
```

Comment:
So, in general we should be using `/* ... */` style comment for copyright and license header. Which also brings up that none of these files have the license header in them.

This should be fixed at some point, but I'm not going to make it an issue for *this* PR.